README.md

```diff
@@ -28,10 +28,12 @@ The Cisco NX-OS collection supports ``network_cli``  and ``httpapi`` connections
 Name | Description
 --- | ---
 [cisco.nxos.nxos](https://github.com/ansible-collections/nxos/blob/master/docs/cisco.nxos.nxos_cliconf.rst)|Use NX-OS cliconf to run commands on Cisco NX-OS platform
+
```

Comment:
**>=2.9,<2.11** need to be changes to **>=2.9.10,<2.11** . right ?

Reply:
Fixed